### PR TITLE
Add subhead to homepage hero. (Fixes #6942)

### LINKED ISSUE #6942: Add subhead to hero section on Mozilla.org for en-US homepage
## Description
Add subhead to hero section on Mozilla.org

The Mozilla.org hero section headline "Live your life. Own your life." is a high-level, aspirational message and by itself may not be enough for visitors to understand what is being offered--a Firefox Account-- and why they might want to click "Learn More".  

LINK TO THE COPY DECK:
https://docs.google.com/document/d/1G3N2o886KrB5kA06OE7ESUKGLW9LwXQBNd8OQxVqJ6A/edit?usp=sharing

### Estimates

| Estimates |  |
| ------------- | ------------- |
| TL | |
| PM | |
| ENG | |
| SEO | |
| ANA | |
| DES | |
| COPY | |

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ]  Subhead is added

#### Tasks
- [ ] Add the new copy found here - https://docs.google.com/document/d/1G3N2o886KrB5kA06OE7ESUKGLW9LwXQBNd8OQxVqJ6A/edit?usp=sharing

### PR DESCRIPTION
**Pull request by amychurchwell**
## Description
Add subhead "Take your privacy and bookmarks everywhere with a Firefox Account." to hero on homepage to provide more context for headline.

## Issue / Bugzilla link
#6942

## Testing
- "Learn more" button has appropriate data-attributes for GA.

### CODE REVIEW

File: bedrock/mozorg/templates/mozorg/home/home-en.html [hunk 1]
```diff
@@ -107,9 +111,7 @@ <h1>{{ self.page_title() }}</h1>
       )}}
 
       <div class="mzp-l-card-half">
-        <!-- 6 -->
```
Comment (stephaniehobson) [inline bedrock/mozorg/templates/mozorg/home/home-en.html]: Is this removal intentional? Or a rebase gone wrong?

Comment (amychurchwell) [reply bedrock/mozorg/templates/mozorg/home/home-en.html]: @stephaniehobson 
It was intentional, I just felt like these comments aren't needed anymore seeing as the page is a lot less verbose and it is now easy to see which card number you are on with the new system.
(ex: `content_card('card_1')`, `content_card('card_2')`, .etc.)

Comment (stephaniehobson) [reply bedrock/mozorg/templates/mozorg/home/home-en.html]: 👍

**Comment by amychurchwell** [comment]
@stephaniehobson Alex mentioned https://github.com/mozilla/bedrock/issues/6942#issuecomment-476253568 that we should consider adding this to the German homepage template. We would need a translated string, which might block this for now?

**Comment by amychurchwell** [comment]
blocked until we get German translation.

**Comment by amychurchwell** [comment]
subhead can be pushed separately on en-us and de. not blocked.

File: bedrock/mozorg/templates/mozorg/home/home-en.html [hunk 2]
```diff
@@ -54,8 +54,8 @@ <h1>{{ self.page_title() }}</h1>
   </header>
 
   {% call download_banner(
-    title=_('Firefox'),
-    sub_title=_('Your life is your business. Not ours.'),
+    logo_title=_('Firefox'),
```
Comment (stephaniehobson) [inline bedrock/mozorg/templates/mozorg/home/home-en.html]: The German page needs to be updated to use the new macro.

File: bedrock/mozorg/templates/mozorg/home/home-en.html [hunk 3]
```diff
@@ -69,11 +69,20 @@ <h1>{{ self.page_title() }}</h1>
     {{ download_firefox(dom_id='download-sticky', download_location='sticky cta', button_color='mzp-t-small') }}
   {% endcall %}
 
-  {{ fxa_banner(
-    title=_('Firefox'),
-    sub_title=_('Live your life.<br> Own your life.'),
-    link_cta=_('Learn More'),
-  )}}
+  {% if l10n_has_tag('mozorg_hero_subhead') %}
```
Comment (stephaniehobson) [inline bedrock/mozorg/templates/mozorg/home/home-en.html]: I think we can do without the l10n_has tag here since the page has not been translated yet. (It's English only right now except for the German page but it gets its own copy).

File: bedrock/mozorg/templates/mozorg/home/home-en.html [hunk 4]
```diff
@@ -69,11 +69,20 @@ <h1>{{ self.page_title() }}</h1>
...
+    {{ fxa_banner(
+      logo_title=_('Firefox'),
```
Comment (stephaniehobson) [inline bedrock/mozorg/templates/mozorg/home/home-en.html]: German page will need the macro parameters updated here too.

File: bedrock/mozorg/templates/mozorg/home/home-de.html [hunk 5]
```diff
@@ -36,9 +36,9 @@ <h1>{{ self.page_title() }}</h1>
   {% endcall %}
 
   {{ fxa_banner(
-    title=_('Firefox'),
-    sub_title=_('Dein Leben. <br>Deine Daten.'),
-    link_cta=_('Weitere Infos'),
+    logo_title=_('Firefox'),
+    title=_('Dein Leben. <br>Deine Daten.'),
+    link_cta=_('Learn More'),
```
Comment (stephaniehobson) [inline bedrock/mozorg/templates/mozorg/home/home-de.html]: Copy-paste missed the Germany CTA. 🍝

Comment (stephaniehobson) [reply bedrock/mozorg/templates/mozorg/home/home-de.html]: This is one of the banners. Does German have the other one? My German is a little rusty* so I'm not sure which one this is.

* I never learned any German.

Comment (amychurchwell) [reply bedrock/mozorg/templates/mozorg/home/home-de.html]: Ah whoops. Yes, will update `download_banner` also. 🍝 😅

**Comment by stephaniehobson** [comment]
Looking good in 🇺🇸 and 🇩🇪 . R+

**Comment by stephaniehobson** [comment]
Happy to get the updated image in here too :)
